### DISCUSSION TITLE
Unable to use student mail.

### DISCUSSION THREAD

**Discussion by notweuz** [community/community#54719] [GitHub Education]
### Hi everyone,

November 2022 was the first time I applied for the Github Education Pack and it was successful, I used my email. Now when I try to reapply, I am told that I need the email address given to us at the school. The problem is that no email address is issued to us, and when I asked the faculty, I was told that even they don't have that address. The address is only given to senior staff members. The college website actually says our email ends with @tempmail.org

**Comment by samus-aran** [comment]
Hi @notweuz 👋 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.
As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient. 

In this case it would be the Academic email(s) which has been successfully associated with both students and faculty at this school.

We would recommend to you speak to your institution regarding to acquiring an email address to proceed.
